src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -4,6 +4,7 @@
 #include "pch.h"
 #include "Tab.h"
 #include "Utils.h"
+#include <debugapi.h>

```

Comment:
Can't merge like this. No debug strings. Use ETL/TraceLog if it needs to remain in production.